### PR TITLE
Add a test RETURNDATACOPY at the beginning of a program should throw …

### PR DESCRIPTION
**Pull request by pirapira**
…an exception if offset + size = 2^256

### CODE REVIEW

**Comment by winsvega** [comment]
you could also check gas usage for throw scenarious

**Comment by pirapira** [comment]
If the gas usage is different, the root state hash will be different.

File: src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_initial_256Filler.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                            "0x00" : "0x01"
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+            "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                "balance" : "0x0de0b6b3a7640000",
+                "code" : "{ (MSTORE 0 0x112233445566778899aabbccddeeff) (RETURNDATACOPY (- 0 32) 0 32) (SSTORE 0 (MLOAD 0)) }",
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_initial_256Filler.json]: if you test the edge condition you could also check 
that test fails   on    X  case.   but it works on X-1 case. and fails on X+1 case. 
you could pass different value through transaction data

Comment (pirapira) [reply src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_initial_256Filler.json]: Added.

File: src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_initial_256Filler.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
         "pre" : {
             "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
                 "balance" : "0x0de0b6b3a7640000",
-                "code" : "{ (MSTORE 0 0x112233445566778899aabbccddeeff) (RETURNDATACOPY (- 0 32) 0 32) (SSTORE 0 (MLOAD 0)) }",
+                "code" : "{ (CALLDATACOPY 0 0 32) (RETURNDATACOPY (- 0 (MLOAD 0)) 0 0x64) (MSTORE 0 0x112233445566778899aabbccddeeff) (SSTORE 0 (MLOAD 0)) }",
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_initial_256Filler.json]: you could also use (CALLDATALOAD 0)

Comment (pirapira) [reply src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_initial_256Filler.json]: I think that's better.

Comment (pirapira) [reply src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_initial_256Filler.json]: Done.
